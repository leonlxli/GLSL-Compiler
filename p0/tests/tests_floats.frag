1.0
1.f
1f
0f
0.1f
0.1
.1
.
.f
+4.5
+4.5f
-4.2f

3foo

1F
1.F
.F
4.5F
-4.2F
+4.5F