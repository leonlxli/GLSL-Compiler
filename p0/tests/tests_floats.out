1.0          line 1 cols 1-3 is T_FloatConstant (value = 1)
1.f          line 2 cols 1-3 is T_FloatConstant (value = 1)
1f           line 3 cols 1-2 is T_FloatConstant (value = 1)
0f           line 4 cols 1-2 is T_FloatConstant (value = 0)
0.1f         line 5 cols 1-4 is T_FloatConstant (value = 0.1)
0.1          line 6 cols 1-3 is T_FloatConstant (value = 0.1)
.1           line 7 cols 1-2 is T_FloatConstant (value = 0.1)
.            line 8 cols 1-1 is '.' 
.f           line 9 cols 1-2 is T_FloatConstant (value = 0)
+            line 10 cols 1-1 is '+' 
4.5          line 10 cols 2-4 is T_FloatConstant (value = 4.5)
+            line 11 cols 1-1 is '+' 
4.5f         line 11 cols 2-5 is T_FloatConstant (value = 4.5)
-            line 12 cols 1-1 is '-' 
4.2f         line 12 cols 2-5 is T_FloatConstant (value = 4.2)
3f           line 14 cols 1-2 is T_FloatConstant (value = 3)
oo           line 14 cols 3-4 is T_Identifier 
1F           line 16 cols 1-2 is T_FloatConstant (value = 1)
1.F          line 17 cols 1-3 is T_FloatConstant (value = 1)
.F           line 18 cols 1-2 is T_FloatConstant (value = 0)
4.5F         line 19 cols 1-4 is T_FloatConstant (value = 4.5)
-            line 20 cols 1-1 is '-' 
4.2F         line 20 cols 2-5 is T_FloatConstant (value = 4.2)
+            line 21 cols 1-1 is '+' 
4.5F         line 21 cols 2-5 is T_FloatConstant (value = 4.5)
